src/main/java/org/jboss/forge/arquillian/command/AddContainerDependencyStep.java

```diff
@@ -96,18 +99,33 @@ public Result execute(UIExecutionContext context) throws Exception {
       Map<Object, Object> ctx = context.getUIContext().getAttributeMap();
       Container container = (Container)ctx.get(ContainerSetupWizard.CTX_CONTAINER);
       String version = (String)ctx.get(ContainerSetupWizard.CTX_CONTAINER_VERSION);
+      Project project = getSelectedProject(context);
+      ArquillianFacet arquillian = project.getFacet(ArquillianFacet.class);
+      ArquillianConfig config = arquillian.getConfig();
 
-      containerInstaller.installContainer(
-            getSelectedProject(context),
-            container,
-            version,
-            getVersionedDependenciesMap());
+      final String profileId = container.getProfileId();
+
+      if (container. isSupportedByChameleon(version)) {
+         dependencyManager.addChameleonDependency(project);
+         if (config.isContainerWithDefault("true")) {
+            config.addContainer(profileId);
+         } else {
+            config.addContainerWithAttribute(profileId, "default", "true");
+         }
```

Comment:
yes you are right. for second call it's done.

no we can't have all qualifier default. Any one qaulifier should be default. If we have multiple qualifier default, then we 'll get exception as `more than one qaulifier is default`